src/playground/sample_data_ingestion/config/ad_plugin_config.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Is there any reason that this file was not in ad_plugin folder ?

Reply:
I have since removed this folder but the tooling looks through all files under `config/` only. The tooling will ignore any directory under `config/`. and thus, any config file were it to be in the said directory.